frigate/api/media.py

```diff
@@ -207,7 +213,41 @@ def get_snapshot_from_recording(camera_name: str, frame_time: str):
     try:
         recording: Recordings = recording_query.get()
         time_in_segment = frame_time - recording.start_time
-        image_data = get_image_from_recording(recording.path, time_in_segment)
```

Comment:
I think it might be better to add these features to this function, otherwise its use isn't consistent

Reply:
This function isn't used anywhere else except in this endpoint. So it could actually be deleted.

Reply:
it is used in the frigate+ endpoint https://github.com/blakeblackshear/frigate/blob/e80322dab75981f8bab3ddaa4667d1cb9ee087fb/frigate/api/media.py#L238